etc/timescaledb/values.yaml

```diff
@@ -39,7 +39,7 @@ auth:
   postgresPassword: ""
   ## @param auth.database Name for a custom database to create
   ##
-  database: grafana-metrics
+  database: data-dashboard
```

Comment:
The database name is being changed here

Reply:
Indeed. Reverted back to `grafana-metrics`